packages/sol/src/session.ts

```diff
@@ -0,0 +1,12 @@
+import { ISession } from "@litespace/types";
+import { randomUUID } from "crypto";
```

Comment:
We cannot use the crypto module in `@litespace/sol`. This package is inteneded to be used in the server, browser, and react native. We don't need to force our ourselves to use polyfills. In this case, please use a third part package that can generate uuids in all enviorments. [uuid](https://www.npmjs.com/package/uuid) can do this.